db/db.go

```diff
@@ -377,7 +377,7 @@ func isSuperORSubset(source, target common.WellFormedName) bool {
 }
 
 func trimBSlash(s string) string {
```

Comment:
It seems this function can be deleted.

```
% grep trimBSlash **/*.go
db/db.go:func trimBSlash(s string) string {
```

Reply:
fixed: https://github.com/vulsio/go-cve-dictionary/pull/439/commits/9a9b25c8618bf47ec5f789c5fdd641edb8ebeb25